ga4gh/cli.py

```diff
@@ -933,6 +828,137 @@ def client_main(parser=None):
             # TODO suppress exception unless debug settings are enabled
             raise exception
 
+
+##############################################################################
+# ga2vcf
+##############################################################################
+
+class Ga2VcfRunner(SearchVariantsRunner):
+    """
+    Runner class for the ga2vcf
+    """
+    def __init__(self, args):
+        super(Ga2VcfRunner, self).__init__(args)
+        self._outputFile = args.outputFile
+        self._binaryOutput = False
+        if args.output_format in "bcf":
```

Comment:
shouldn't this be `args.output_format == "bcf"`?  similar comment for a number of other places where `output_format` is used